<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,4,0,1">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Список автомобилей</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tw_vehicle">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Список маршрутов</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tw_routes"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pb_create_vehicle">
         <property name="text">
          <string>Создать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_update_vehicle">
         <property name="text">
          <string>Обновить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_delete_vehicle">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Модель</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_model"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Регистрационный номер</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_plate"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Год выпуска</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_year">
         <property name="minimum">
          <number>1900</number>
         </property>
         <property name="maximum">
          <number>2100</number>
         </property>
         <property name="value">
          <number>2000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Грузоподъёмность</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_payload">
         <property name="maximum">
          <number>50000</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Пробег</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_run">
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Расход топлива</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dsb_fuel_consumption">
         <property name="decimals">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Объём кузова</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dsb_volume">
         <property name="decimals">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
